### DISCUSSION TITLE
Welcome to Amazon-Flipkart-Price-Comparison-Engine Discussions!

### DISCUSSION THREAD

**Discussion by sushantPatrikar** [sushantPatrikar/Amazon-Flipkart-Price-Comparison-Engine#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hsnyderburn** [comment]
Hi,

Sorry if I missed this somewhere, but what version of python did you use for this?

_- found details for packages in the requirements.txt_
